packages/chart/package.json

```diff
@@ -1,84 +1,94 @@
-    "@types/d3-transition": "1.1.4",
-    "d3-array": "1.2.4",
-    "d3-axis": "1.0.8",
-    "d3-brush": "1.0.6",
-    "d3-color": "1.2.3",
-    "d3-contour": "1.1.2",
-    "d3-format": "1.3.2",
-    "d3-geo": "1.6.4",
-    "d3-hexbin": "0.2.2",
-    "d3-hierarchy": "1.1.4",
-    "d3-hsv": "0.1.0",
-    "d3-interpolate": "1.3.2",
-    "d3-scale": "1.0.7",
-    "d3-selection": "1.4.0",
-    "d3-shape": "1.2.0",
-    "d3-svg-annotation": "2.2.2",
-    "d3-time-format": "2.1.3",
-    "d3-transition": "1.2.0",
-    "d3-zoom": "1.8.3",
-    "d3v4-bullet": "1.0.6",
-    "npm-run-all": "4.1.5",
-    "rimraf": "2.6.3",
-    "rollup": "1.20.2",
-    "rollup-plugin-alias": "2.0.0",
-    "rollup-plugin-commonjs": "10.1.0",
-    "rollup-plugin-node-resolve": "5.2.0",
-    "rollup-plugin-postcss": "2.0.3",
-    "rollup-plugin-sourcemaps": "0.4.2",
-    "terser": "4.0.0",
-    "tslib": "1.10.0",
-    "tslint": "5.17.0",
-    "typedoc": "0.14.2",
-    "typescript": "3.6.4"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/hpcc-systems/Visualization.git"
-  },
-  "author": "Gordon Smith <gordonjsmith@gmail.com>",
-  "contributors": [],
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/hpcc-systems/Visualization/issues"
-  },
-  "homepage": "https://github.com/hpcc-systems/Visualization",
-  "gitHead": "90532e4914f063f90c74dbb392c074f783440017"
-}
+    "name": "@hpcc-js/chart",
+    "version": "2.33.0",
+    "description": "hpcc-js - Viz Chart",
+    "main": "dist/index.js",
+    "module": "dist/index.es6",
+    "unpkg": "dist/index.min.js",
+    "jsdelivr": "dist/index.min.js",
+    "types": "types/index.d.ts",
+    "typesVersions": {
+        "<3.8": {
+            "*": [
+                "types-ts3.4/*"
+            ]
+        }
+    },
+    "files": [
+        "dist/*",
+        "types/*",
+        "types-3.4/*",
+        "src/*.css"
+    ],
+    "scripts": {
+        "clean": "rimraf lib* types dist *.tsbuildinfo",
+        "compile-es6": "tsc --module es6 --outDir ./lib-es6",
+        "compile-es6-watch": "npm run compile-es6 -- -w",
+        "compile-umd": "tsc --module umd --outDir ./lib-umd",
+        "compile-umd-watch": "npm run compile-umd -- -w",
+        "bundle": "rollup -c",
+        "bundle-watch": "npm run bundle -- -w",
+        "minimize": "terser dist/index.js -c -m --source-map \"content='dist/index.js.map',url='index.min.js.map'\" -o dist/index.min.js",
+        "gen-legacy-types": "downlevel-dts ./types ./types-3.4",
+        "build": "npm run compile-es6 && npm run bundle",
+        "watch": "npm-run-all compile-es6 -p compile-es6-watch bundle-watch",
+        "stamp": "node node_modules/@hpcc-js/bundle/src/stamp.js",
+        "lint": "tslint --project . src/**/*.ts",
+        "docs": "typedoc --options tdoptions.json ."
+    },
+    "dependencies": {
+        "@hpcc-js/api": "^2.8.20",
+        "@hpcc-js/common": "^2.26.0",
+        "@hpcc-js/util": "^2.14.0"
```

Comment:
Lowered the version here? Was that intentional?

Reply:
This will get auto "corrected" during the next release cycle.